pkg/validation/app_test.go

```diff
@@ -142,7 +142,10 @@ func Test_ValidateApp(t *testing.T) {
 					Version: "1.2.2",
 				},
 			},
-			expectedErr: "validation error: catalog `control-plane-catalog` not found",
+			catalogs: []*v1alpha1.Catalog{
+				newTestCatalog("control-plane-catalog", "giantswarm"),
```

Comment:
If you added an empty string here "", would it return a validation error?

Reply:
No the test still passes if you use empty strings.

I think its because the tests use mock objects. So the openapi validation rules in the Catalog CRD are not checked.